velox/common/caching/SsdFile.cpp

```diff
@@ -652,12 +671,14 @@ void SsdFile::initializeCheckpoint() {
   std::ifstream state(fileName_ + kCheckpointExtension);
   if (!state.is_open()) {
     hasCheckpoint = false;
+    ++stats_.errorsOpenCheckpoint;
     LOG(INFO) << "Starting shard " << shardId_ << " without checkpoint";
   }
   auto logPath = fileName_ + kLogExtension;
   evictLogFd_ = open(logPath.c_str(), O_CREAT | O_RDWR, S_IRUSR | S_IWUSR);
   if (evictLogFd_ < 0) {
     // Failure to open the log at startup is a process terminating error.
+    ++stats_.errorsOpenLog;
```

Comment:
The program will exit here so no need to add stats here. Can you change the error log to a VELOX_CHECK_NE as @spershin did for cowFile setting? Thanks!

Reply:
Revised.

Reply:
```
s/errorsOpenLog/openCheckpointErrors/
```